File: php/tests/Pe1Test.php
<?php

namespace Tests;

use Pe\Pe1;
use PHPUnit\Framework\TestCase;

class Pe1Test extends TestCase
{
    public function testCalc()
    {
        $this->assertEquals(234168, Pe1::with_loop(1000));
        $this->assertEquals(234168, Pe1::no_loop(1000));
    }
}

File: cpp/pe/test/pe1_test.cc
#include "gtest/gtest.h"
#include "../include/pe1.h"

namespace {
  TEST(Pe1Test, Main) {
    EXPECT_EQ(234168, pe::with_loop(1000));
    EXPECT_EQ(234168, pe::no_loop(1000));
  }
}